firebase-admin/runtime/pom.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <groupId>io.quarkiverse.googlecloudservices</groupId>
+        <artifactId>quarkus-google-cloud-firebase-admin-parent</artifactId>
+        <version>1.3.0-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>quarkus-google-cloud-firebase-admin</artifactId>
+    <name>Quarkus - Google Cloud Services - Firebase Admin - Runtime</name>
+    <description>Use Google Cloud Firebase Admin to perform privileged actions</description>
+
+    <properties>
+        <firebase-admin-sdk.version>9.1.0</firebase-admin-sdk.version>
+    </properties>
```

Comment:
Is it needed ?
Most GCP related versions are managed by the Google Cloud Library BOM.

Reply:
Unfortunately, the BOM does not include the Admin SDK. 😞

Reply:
OK, good for me